### PR TITLE
Setup angular

### PR DESCRIPTION
**Pull request by yeeteing**
- create an angular application in /OnLand/ControlLaptop/ directory
- added instruction on how to setup the application (In readme)
- configured the application to ease the development process in the future: 
Screenshot_2020-11-01_19-50-07

### CODE REVIEW

**Comment by colton-smith** [comment]
Initial comment:

One small thing to look out for is branch naming rules, they are outlined here: https://sites.google.com/mun.ca/paradigmsoftwarewiki/guides/git-workflow?authuser=0

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+## Front-end (Angular) setup Guide
+1. First, open your terminal. Navigate to `/comp1-software` directory.
+2. Run `npm install -g @angular/cli`
+3. Navigate to `/OnLand/ControlLaptop/angular-UI`. 
+    If you are in the root of the 'comp1-software' folder, then just run:
+        Windows: `cd OnLand\ControlLaptop\angular-UI`
+        Linux/Mac: `cd OnLand/ControlLaptop/angular-UI`
+4. When you in the angular-UI folder, run `npm install`
+5. To build and serves the app locally, run `ng serve`. If it's build successfully, there should be a link that look like this: http://localhost:4200/. 
+6. Navigate to the link in your browser, and your should be able to see the app.
```
Comment (colton-smith) [inline README.md:13]: I really appreciate you adding Windows and Linux notes, I tend to add Linux ones and then people on windows are like dude what is this...

Comment (yeeteing) [reply README.md:13]: :smile:

**Comment by colton-smith** [comment]
```
csmith@LAPTOP-HVCPJH5T:/mnt/c/paradigm/boring/comp1/comp1-software/OnLand/ControlLaptop/angular-UI$ npm install

added 1496 packages, and audited 1496 packages in 6m

4 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
```

@yeeteing Are these messages normal for angular? Just caused by angular using older versions of a few packages?

**Review by colton-smith — APPROVED** [review]
After following this guide to install Node Version Manager and then node/npm: 
https://docs.microsoft.com/en-us/windows/nodejs/setup-on-wsl2

 I was able to follow the notes in the readme and serve the development angular server locally,
AKA -> it worked!

GTM!

**Comment by yeeteing** [comment]
> After following this guide to install Node Version Manager and then node/npm:
> https://docs.microsoft.com/en-us/windows/nodejs/setup-on-wsl2
> 
> I was able to follow the notes in the readme and serve the development angular server locally,
> AKA -> it worked!
> 
> GTM!

haha That remind me to make a note on "(prerequisite)  install Nodejs"

**Review by colton-smith — APPROVED** [review]
Updated readme looks good, you can go ahead and merge this now.

**Pull request merged**